variant_to_realignment.nf

```diff
@@ -193,17 +227,30 @@ process readsToRemappedVariants {
     input:
         path "reads_aligned.bam"
         path "genome.fa"
+        val flank_length
+        val filter_align_with_secondary
 
     output:
         path "variants_remapped.vcf", emit: variants_remapped
         path "variants_unmapped.vcf", emit: variants_unmapped
+        path "summary.yml", emit: summary_yml
 
+    script:
+    if (filter_align_with_secondary)
     """
     # Ensure that we will use the reads_to_remapped_variants.py from this repo
     ${baseDir}/variant_remapping_tools/reads_to_remapped_variants.py -i reads_aligned.bam \
-        -o variants_remapped.vcf  --filter_align_with_secondary \
-        --newgenome genome.fa --out_failed_file variants_unmapped.vcf
+        -o variants_remapped.vcf  --newgenome genome.fa --out_failed_file variants_unmapped.vcf \
+        --flank_length $flank_length --summary summary.yml --filter_align_with_secondary
+    """
```

Comment:
missed this on a previous PR but these should probably be indented

Reply:
Done